src/patterns/phone-numbers/index.njk

```diff
@@ -0,0 +1,119 @@
+---
+title: Phone numbers
+group: Ask users for...
+---
+{% from "views/partials/example/_macro.njk" import patternlibExample %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% from "components/external-link/_macro.njk" import onsExternalLink %}
+{% from "components/lists/_macro.njk" import onsList %}
+
+
+How to ask users for a phone number.
+
+{{
+    patternlibExample({"path": "components/input/examples/telephone/index.njk"})
+}}
+
+## When to use this pattern
+Only ask users for a phone number when your service has a genuine need for this information. For example, to send an access code by text message. 
+
+## How to use this pattern
+You should always explain why you need the user's phone number, letting them know if you will contact them and when.
+
+Use an [input component](/components/input) and set the input type to `tel` to bring up the correct keypad on touch devices.
+
+When using this pattern you should:
+{{
+    onsList({
+        "itemsList": [
+            {
+                "text": 'tell users why you need a phone number and what it will be used for'
+            },
+            {
+                "text": 'make it clear what type of telephone number you need'
+            },
+            {
+                "text": 'make sure you allow the user to enter a phone number in a familiar format'
+            },
+            {
+                "text": 'help users enter a phone number in the correct format'
+            }
+        ]
+    })
+}}
+
+### Explain why you need the phone number
+Use the optional label__description with the [input component](/components/input) for hint text to reassure users why you need a phone number and what it will be used for.
+
+## Make it clear what type of phone number you need
+Use the `label__description` for hint text to let users know what type of phone number you need, for example, a UK, international or mobile phone number.
+
+## Allow all phone number formats
+The phone number field must accommodate all phone number formats. The length of the field must allow for any spaces or common characters, for example, brackets, dashes or country codes.
```

Comment:
@LauraC119 Think this could do with mentioning the user.  E.g. 
```suggestion
The phone number field must accommodate all phone number formats so the user can enter the number in a format they’re familiar with. The length of the field must allow for any spaces or common characters, for example, brackets, dashes or country codes.
```

Reply:
I see your point but why did we decide not to include the "familiar" bit that's in the bullet list above? I'm sure we had it there but took it out! But I agree that it helps. Change slightly, though, so we avoid that hanging "with" and don't use "format" twice:
The phone number field must accommodate all phone number variations so the user can enter a number in the format they are used to.